src/Backend/Core/Layout/Templates/head.html.twig

```diff
@@ -8,33 +8,20 @@
   <link rel="shortcut icon" href="/src/Backend/favicon.ico" />
 
   <script defer data-search-pseudo-elements src="https://use.fontawesome.com/releases/v5.11.2/js/all.js" integrity="sha384-b3ua1l97aVGAPEIe48b4TC60WUQbQaGi2jqAWM90y0OZXZeyaTCWtBTKtjW2GXG1" crossorigin="anonymous"></script>
-  <link href='https://fonts.googleapis.com/css?family=Source+Code+Pro%7COpen+Sans:400,600' rel='stylesheet' type='text/css'>
 
-  {% for cssFile in cssFiles %}
-    <link rel="stylesheet" href="{{ cssFile }}" />{{ CRLF }}{{ TAB }}
-  {% endfor %}
+  <link href='https://fonts.googleapis.com/css?family=Source+Code+Pro%7COpen+Sans:400,600' rel='stylesheet' type='text/css'>
 
   {% for jsFile in jsFiles %}
     <script src="{{ jsFile }}"></script>{{ CRLF }}{{ TAB }}
   {% endfor %}
 
+
+  {{ encore_entry_script_tags('backend', null, 'backendConfig') }}
+  {{ encore_entry_link_tags('screen', null, 'backendConfig') }}
+
   {% if debug %}
-    <script src="http://localhost:35729/livereload.js?snipver=1"></script>
+    <script src="http://localhost:35729/livereload.js"></script>
```

Comment:
When I develop css/js for Fork themes, I use webpack dev server with hot module reloads, is that something we can use ? Not sure if it's been looked into https://symfony.com/doc/current/frontend/encore/dev-server.html

Reply:
I have looked at the hot module reloads but could not find a way to make it work, so I choose to do it this way. But feel free to upgrade it!